vector/build.gradle

```diff
@@ -508,4 +508,5 @@ dependencies {
     androidTestImplementation('com.adevinta.android:barista:4.1.0') {
         exclude group: 'org.jetbrains.kotlin'
     }
+    androidTestUtil 'androidx.test:orchestrator:1.4.0'
```

Comment:
For dependency common in several module, we should now use [this file](https://github.com/vector-im/element-android/blob/develop/dependencies.gradle), introduced by @ariskotsomitopoulos.